workerapi/client/client.go

```diff
@@ -134,12 +133,19 @@ func (c *simpleClient) StatusAll() ([]runner.RunStatus, error) {
 	return st.Runs, nil
 }
 
+// Implements Scoot Worker API
 func (c *simpleClient) QueryNow(q runner.Query) ([]runner.RunStatus, error) {
```

Comment:
While we are in here, can we change the name to be consistent with our other run status retrieval apis (right now we have Status, StatusAll and QueryNow).... can we change QueryNow to FilteredStatus (or something with the word Status in it)